pennylane/transforms/zx/converter.py

```diff
@@ -0,0 +1,302 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Transforms for interacting with PyZX, framework for ZX calculus."""
+# pylint: disable=too-many-statements, too-many-branches
+
+from collections import OrderedDict
+import numpy as np
+import pennylane as qml
+from pennylane.tape import QuantumTape
+from pennylane.wires import Wires
+
+
+def to_zx(tape, backend=None):
+    """It converts a PennyLane quantum tape to a ZX-Graph in PyZX.
+    Args:
+        tape(QuantumTape): The PennyLane quantum tape.
```

Comment:
```suggestion
        qscript (QuantumScript): The PennyLane quantum script.
```
We are trying to remove all references to `tape` in PennyLane.